### ISSUE TITLE
6.8.0:Test file makes project build fail

### ISSUE THREAD

**Issue by NiuGuohui** [wix/react-native-ui-lib#1790]
The new version brings the test file into src/index.js.When I install it, an error is reported.

```shell
error: Error: Unable to resolve module @testing-library/react-native from /xxxxxxxxx/node_modules/react-native-ui-lib/src/components/text/Text.driver.js: @testing-library/react-native could not be found within the project or in these directories:
  node_modules/react-native-ui-lib/node_modules
  node_modules

If you are sure the module exists, try these steps:
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*

```

### Related to

- [x] Components
- [ ] Docs
- [ ] Typings
- [ ] Demo

### Code snippet

```


```

### Environment

- React Native:
- React Native UI Lib:

### Affected platforms

- [ ] Android
- [ ] iOS
- [ ] Web

**Comment by NiuGuohui** [comment]
I had to manually install @testing-library/react-native to fix the error.

**Comment by ethanshar** [comment]
We will take a look and release a HF

**Comment by ethanshar** [comment]
Fixed in 6.8.1

**Comment by NiuGuohui** [comment]
> Fixed in 6.8.1

thanks
